### PR TITLE
Fix logging of wrong feature_columns in CLI and exception in auto-complete

### LINKED ISSUE #819: [BUG]: Morpheus CLI logged feature_columns don't match actual features
### Version

23.03

### Which installation method(s) does this occur on?

Docker, Conda, Source

### Describe the bug.

Not all pipelines set `feature_columns` in the pipeline config, specifically the `ransomware_detection` and `abp_pcap_detection` examples don't set these. 

When we run these with their associated `run.py` script this isn't a problem, however using the equivalent CLI command the default features for the FIL pipeline are logged (the nvsmi features).

I think we have two options:
1) Simply don't log the `config.fil.feature_columns` and `config.ae.feature_columns`.
2) Re-work these examples to populated the config.

Personally I like option 1 

### Minimum reproducible example

_No response_

### Relevant log output

_No response_

### Full env printout

_No response_

### Other/Misc.

_No response_

### Code of Conduct

- [X] I agree to follow Morpheus' Code of Conduct
- [X] I have searched the [open bugs](https://github.com/nv-morpheus/Morpheus/issues?q=is%3Aopen+is%3Aissue+label%3Abug) and have found no duplicates for this bug report

### LINKED ISSUE #984: [BUG]: CLI auto-completion attempts to load plugins which are already loaded
### Version

23.07

### Which installation method(s) does this occur on?

Source

### Describe the bug.

This occurs when a user presses the tab key on the CLI when building a pipeline and there is at least one stage already in the command resulting in a `RuntimeError`.

### Minimum reproducible example

```shell
morpheus run pipeline-nlp from-file
```
Then press the tab key.

### Relevant log output

```shell
$ morpheus run pipeline-nlp from-file Traceback (most recent call last):
  File "/home/dagardner/work/conda/envs/morpheus/bin/morpheus", line 8, in <module>
    sys.exit(run_cli())
  File "/home/dagardner/work/morpheus/morpheus/cli/run.py", line 24, in run_cli
    cli(obj={}, auto_envvar_prefix='MORPHEUS', show_default=True, prog_name="morpheus")
  File "/home/dagardner/work/conda/envs/morpheus/lib/python3.10/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/home/dagardner/work/conda/envs/morpheus/lib/python3.10/site-packages/click/core.py", line 1050, in main
    self._main_shell_completion(extra, prog_name, complete_var)
  File "/home/dagardner/work/conda/envs/morpheus/lib/python3.10/site-packages/click/core.py", line 1125, in _main_shell_completion
    rv = shell_complete(self, ctx_args, prog_name, complete_var, instruction)
  File "/home/dagardner/work/conda/envs/morpheus/lib/python3.10/site-packages/click/shell_completion.py", line 49, in shell_complete
    echo(comp.complete())
  File "/home/dagardner/work/conda/envs/morpheus/lib/python3.10/site-packages/click/shell_completion.py", line 291, in complete
    completions = self.get_completions(args, incomplete)
  File "/home/dagardner/work/conda/envs/morpheus/lib/python3.10/site-packages/click/shell_completion.py", line 273, in get_completions
    return obj.shell_complete(ctx, incomplete)
  File "/home/dagardner/work/conda/envs/morpheus/lib/python3.10/site-packages/click/core.py", line 1438, in shell_complete
    results.extend(super().shell_complete(ctx, incomplete))
  File "/home/dagardner/work/conda/envs/morpheus/lib/python3.10/site-packages/click/core.py", line 957, in shell_complete
    results.extend(
  File "/home/dagardner/work/conda/envs/morpheus/lib/python3.10/site-packages/click/core.py", line 957, in <genexpr>
    results.extend(
  File "/home/dagardner/work/conda/envs/morpheus/lib/python3.10/site-packages/click/core.py", line 59, in _complete_visible_commands
    for name in multi.list_commands(ctx):
  File "/home/dagardner/work/morpheus/morpheus/cli/commands.py", line 111, in list_commands
    raise RuntimeError("Plugins registered the following duplicate commands: {}".format(
RuntimeError: Plugins registered the following duplicate commands: from-file
```

### Full env printout

_No response_

### Other/Misc.

_No response_

### Code of Conduct

- [X] I agree to follow Morpheus' Code of Conduct
- [X] I have searched the [open bugs](https://github.com/nv-morpheus/Morpheus/issues?q=is%3Aopen+is%3Aissue+label%3Abug) and have found no duplicates for this bug report

### PR DESCRIPTION
**Pull request by dagardner-nv**
## Description
The CLI command for the FIL pipeline set a default value for `--columns_file` flag but some FIL pipelines like ransomeware didn't use this config property, causing the default columns file to be loaded and for the values to be logged to the terminal.

* Removed default values for `--columns_file` and `--labels-file` flags
* Removed redundant checks for null values if they were already being enforced by `required=True`
* Fix raised exception when using click auto-complete
* Misc pylint/flake8 errors

fixes #984
fixes #819

## Checklist
- [x] I am familiar with the [Contributing Guidelines](https://github.com/nv-morpheus/Morpheus/blob/main/docs/source/developer_guide/contributing.md).
- [x] New or existing tests cover these changes.
- [x] The documentation is up to date with these changes.

### CODE REVIEW

**Review by mdemoret-nv — APPROVED** [review]

**Comment by mdemoret-nv** [comment]
Waiting on review from @pdmack

**Review by pdmack — APPROVED** [review]
LGTM

**Pull request merged**
